styles.css

```diff
@@ -0,0 +1,413 @@
+* {
```

Comment:
This selector should be `*, *::before, *::after`. Take a read about it [here](https://www.paulirish.com/2012/box-sizing-border-box-ftw/)